templates/applicantForm.phtml

```diff
@@ -18,50 +18,50 @@
             <div id="whyDevError" class="alert hidden formItem_alert"></div>
-            <textarea id="why-developer" name="whyDev" class="submitApplicant form-control" placeholder="Enter here" required></textarea>
+            <textarea id="why-developer" name="whyDev" class="submitApplicant form-control" placeholder="Enter here" required><?php echo $applicantWhyDev ?></textarea>
 
             <label for="experience">Please tell us about any relevant coding experience.</label>
             <div id="codeExperienceError" class="alert hidden formItem_alert"></div>
-            <textarea id="experience" name="codeExperience" class="submitApplicant form-control" placeholder="Enter here"></textarea>
+            <textarea id="experience" name="codeExperience" class="submitApplicant form-control" placeholder="Enter here"><?php echo $applicantCodeExperience ?></textarea>
 
             <label for="hear-about">How did you hear about Mayden Academy?</label>
-            <select id="hear-about" class="submitApplicant form-control" name="hearAboutId" required>
+            <select id="hear-about" class="submitApplicant form-control" name="hearAboutId" data-selected="<?php echo $applicantHearAboutId ?>" required>
             </select>
 
             <div class="form-group">
-                <input class="submitApplicant" name="eligible" type="checkbox" />
+                <input class="submitApplicant" name="eligible" type="checkbox" <?php echo $applicantEligible ? "checked" : ""; ?>/>
                 <label for="uk-eligible">I confirm I am eligible to study in the UK.</label>
             </div>
 
             <div>
-                <input class="submitApplicant" name="eighteenPlus" type="checkbox" />
+                <input class="submitApplicant" name="eighteenPlus" type="checkbox" <?php echo $applicantEighteenPlus ? "checked" : ""; ?>/>
                 <label for="is-18">I confirm I will be at least 18 years before my chosen cohort*.</label>
             </div>
 
             <div class="form-group">
-                <input class="submitApplicant" name="finance" type="checkbox" />
+                <input class="submitApplicant" name="finance" type="checkbox" <?php echo $applicantFinance ? "checked" : ""; ?>/>
                 <label for="finance">I would like to apply for interest free finance.</label>
             </div>
 
-            <label for="notes">Notes</label>
+            <label for="notes">Notes.</label>
             <div id="notesError" class="alert hidden formItem_alert"></div>
-            <textarea id="notes" name="notes" class="submitApplicant form-control" placeholder="Enter here"></textarea>
+            <textarea id="notes" name="notes" class="submitApplicant form-control" placeholder="Enter here"><?php echo $applicantNotes ?></textarea>
 
             <input id="submitApplicant" type="submit" class="btn btn-success"/>
             <div id="generalError" class="alert alert-danger hidden formItem_alert"></div>
```

Comment:
there is also a go back to the admin page button but no return to applicants list button which doesnt make sense. again, need to feed this data in from the controller